docs/usage/configuration/user_authentication/refresh_tokens.md

```diff
@@ -0,0 +1,124 @@
+# Refresh Tokens
+
+Synapse supports refresh tokens since version 1.49 (some earlier versions had support for an earlier, experimental draft of [MSC2918] which is not compatible).
+
+
+[MSC2918]: https://github.com/matrix-org/matrix-doc/blob/main/proposals/2918-refreshtokens.md#msc2918-refresh-tokens
+
+
+## Background and motivation
+
+Synapse users' sessions are identified by **access tokens**; access tokens are
+issued to users on login and the access token is then used to identify the user
+and device making subsequent requests.
+
+Traditionally, these access tokens were eternally valid (at least until the user
+explicitly chose to log out).
+
+In some cases, it may be desirable for these access tokens to expire so that the
+potential damage caused by leaking an access token is reduced.
+On the other hand, forcing a user to re-authenticate (log in again) often might
+be too much of an inconvenience.
+
+**Refresh tokens** are a mechanism to avoid some of this inconvenience whilst
+still getting most of the benefits of short access token lifetimes.
```

Comment:
It's not quite OAuth/OIDC (afaik) since my experience with those shows that they use a slightly different format, but the principle is the same. Perhaps I can indeed link to that, saying the principle is the same.

Reply:
s/based on/inspired by?

Reply:
was more just thinking of a link to prior art, to provide context and make it clear that this isn't a synapse-specific concept